### DISCUSSION TITLE
Support memory-only firestore build

### DISCUSSION THREAD

**Discussion by swftvsn** [angular/angularfire#2370] [Ideas]
Support was added in 7.13.0 (https://firebase.google.com/support/release-notes/js#cloud-firestore_1) of Firebase JavaScript SDK. The import should be in that case 

`import 'firebase/firestore/memory';` instead of 
`import 'firebase/firestore';` 

Not sure what is the best approach to support conditional imports? Anyhow, this is something we'd like to have, as we prefer (due to large amounts of data) to skip the persistence totally.

**Comment by jamesdaniels** [comment]
My plan is to support it in AngularFire 7.1 in a lazy-loaded version of `AngularFirestore`.

**Comment by swftvsn** [comment]
Awesome! This is looking more and more great every release :D I'll take it that you mean 6.1 though?
